postgres/client.js

```diff
@@ -207,15 +213,23 @@ function del(key) {
  * @param  {[type]} ops [description]
  * @return {[type]}     [description]
  */
+/* eslint complexity: 0 */
 function batch(ops) {
   var commands = [], url;
 
-  for (let i = 0; i < ops.length; i++) {
+  for (let i = 0, opsLength = ops.length; i < opsLength; i++) {
```

Comment:
Why are we using `opsLength` here and not `ops.length`?

Reply:
This is done for loop optimization, we assign the value of `length` to a property to avoid having to look it up at every iteration. You can read more about it here: https://jaysoo.ca/2009/12/23/javascript-optimizing-loops/